src/modules/dashboard/components/ExpenditurePage/LogsSection/LogsSection.css

```diff
@@ -0,0 +1,105 @@
+@value inputBackground: rgb(249, 250, 250);
+@value shadow: inset 0px 2px 4px color-mod(black alpha(7%));
+
+.container {
+  padding: 0 25px;
+}
+
+.comment div {
+  padding: 0;
+  font-size: var(--size-tiny);
+  font-weight: var(--weight-bold);
+}
+
+.comment > div > div {
+  width: 58px;
+}
+
+.commentInput {
+  margin-top: 29px;
+}
+
+.commentInput textarea::placeholder {
+  font-size: var(--size-smallish);
+  font-style: normal;
+  color: var(--text-disabled);
+}
+
+.disabledComment {
+  margin-top: 28px;
+}
+
+.disabledComment input {
+  padding: 11px 8px;
+  height: 40px;
+  border: 1px solid var(--temp-grey-blue-4);
+  border-radius: var(--radius-tiny);
+  background-color: inputBackground;
+  font-size: var(--size-smallish);
+  font-weight: var(--weight-normal);
+  line-height: var(--size-medium-l);
+  box-shadow: shadow;
+}
+
+.disabledComment input::placeholder {
+  font-style: normal;
+  color: var(--text-disabled);
```

Comment:
This is not needed anymore, right? As you are using the `CommentInput` component.